### DISCUSSION TITLE
post/2023-03-20-comments/

### DISCUSSION THREAD

**Discussion by giscus** [marcusramberg/blog#8] [Announcements]
# post/2023-03-20-comments/

Making (things|code|mischief|mistakes).

https://marcus.means.no/post/2023-03-20-comments/

**Comment by denisbr** [comment]
Bah, this is never going to work, and all your comments will be trollz!!1!

**Reply by marcusramberg** [reply]
I guess you're right. Everything is hopeless and we're all gonna die :-/
